diff --git a/src/lib/stripe.ts b/src/lib/stripe.ts
index c80edab..981faa2 100644
--- a/src/lib/stripe.ts
+++ b/src/lib/stripe.ts
@@ -6,7 +6,7 @@ let stripeClient: Stripe | undefined;
 export function getStripe() {
   const apiKey = process.env.STRIPE_SECRET_KEY;
   if (!apiKey) throw new Error("Stripe is not configured");
-  stripeClient ??= new Stripe(apiKey, { apiVersion: "2026-08-26.dahlia" });
+  stripeClient ??= new Stripe(apiKey, { apiVersion: "2026-07-29.dahlia" });
   return stripeClient;
 }
 
